RPCServer/rpcwallet.py

```diff
@@ -48,8 +50,15 @@ def __init__(self, path='.wallet', verbose=False):
         self.load()
         if self.verbose:
             print(self.index)
-                
-                
+
+
+    def _is_json(self, json_string):
```

Comment:
Means we're reading the file twice.
don't. Just do a 
if ".json" in file
to id .json , then wrap a single json.load within a try..except to deal with potential broken files.

No need for this method then.